test/doc/core/regular/abs.cpp

```diff
@@ -12,10 +12,8 @@ int main()
 
   std::cout << "---- simd" << '\n'
             << "<- pf                  = " << pf << '\n'
-            
             << "-> abs(pf)             = " << eve::abs(pf) << '\n'
             << "<- pi                  = " << pi << '\n'
-            << "-> saturated(abs)(pi)  = " << eve::saturated(eve::abs)(pi) << '\n'
```

Comment:
why delete saturated from abs docs?

Reply:
the doc include this one AND the saturated doc, so it was redudant